### PR TITLE
added a div

### PR DESCRIPTION
**Pull request by alexanderyanthar**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 <body>
     <h1>This is a prTest</h1>
     <p>who wants to style anyway?</p>
+    <div>I've added somethign new</div>
```
Comment (erikajdavid) [inline index.html:11]: does this need a class name?

**Review by erikajdavid — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (alexanderyanthar) [reply index.html:11]: yaaa

File: index.html [hunk 2]
```diff
@@ -8,5 +8,6 @@
 <body>
     <h1>This is a prTest</h1>
     <p>who wants to style anyway?</p>
```
Comment (alexanderyanthar) [inline index.html:10]: I want to test the start a review button

Comment (alexanderyanthar) [reply index.html:10]: I guess that's all

**Review by alexanderyanthar — COMMENTED** [review]
coolio

File: index.html (hunk 2 above)
Comment (erikajdavid) [reply index.html:10]: hello

**Review by erikajdavid — APPROVED** [review]

**Pull request merged**